docs/source/conf.py

```diff
@@ -64,7 +65,13 @@
 templates_path = ['_templates']
 
 # always execute notebooks
-nbsphinx_execute = 'always'
+env_skip_execute = os.getenv("SKIP_EXECUTE")
+
+if env_skip_execute is None:
```

Comment:
This will let me set `SKIP_EXECUTE=TRUE` in my shell by default, then run `SKIP_EXECUTE= make html` to opt-in to running the notebooks
```suggestion
if env_skip_execute:
```